src/Appium.Net/Appium/Android/AndroidDriver.cs

```diff
@@ -22,7 +22,6 @@
 using System.Drawing;
 using System.Linq;
 using OpenQA.Selenium.Appium.Android.Enums;
-using OpenQA.Selenium.Appium.Interactions;
```

Comment:
I assume the IDE should have instruments that automatically remove unused imports from all files at once

Reply:
No, but i'll run resharper to do so on the solution